src/aws_locker.py

```diff
@@ -110,8 +110,9 @@ def load_profiles(cred_lines):
 
     # validate all profiles have correct entries, delete ones without
     for profile in profiles:
-        if "aws_access_key_id" not in profiles[profile] or \
```

Comment:
this is looking for aws_access_key_id twice which is wrong... however the fix should not have converted the "or" to an "and" since if either one is missing we would like to throw an error.